packages/@aws-cdk/aws-logs/README.md

```diff
@@ -119,6 +114,22 @@ new logs.SubscriptionFilter(this, 'Subscription', {
 });
 ```
 
+If your `SubscriptionFilter` destination lives in a different account, a `CrossAccountDestination`
+object needs to be added in the destination account which will act as a proxy
+for the remote resource. After you have created the `CrossAccountDestination` in the destination account,
+use the `CrossAccountDestination.fromDestinationArn` method in the source account to import the destination:
```

Comment:
```
This object is automatically created for you
if you use the CDK Kinesis library.
```

This text has been deleted, but these changes don't indicate that the behavior will. It's worth mentioning a list of resources which will automatically create the destination for you, even if it's just Kinesis.

Reply:
From what I can find, this behavior doesn't currently exist. It was added in https://github.com/aws/aws-cdk/commit/9d58d7a91a8ad06d7460c408733ac54ebc796a6e#diff-91d106241cb46018eea8934193bfcab27e75c19393ea7472382bccf5c776e8ba and removed in https://github.com/aws/aws-cdk/commit/01601c2c9218da573d0c94f72ea0639551a7160b#diff-04dca47d05b6466108d9f1a1742ddbfc5544fe3221e3efde3741a928d0ee4565.

The link between Kinesis and CloudWatch was shifted to the `@aws-cdk/aws-logs-destinations` package, and these docs weren't updated at the time.